src/components/Navbar.css

```diff
@@ -1,9 +1,13 @@
 .navbar {
     display: flex;
+<<<<<<< HEAD
```

Comment:
remove this section i.e

<<<<<<< HEAD
    position: relative;
=======
>>>>>>> main

its part of merge conflict

Reply:
done !! now is it perfect ?

Reply:
can you push that change? cant see rn here..

Reply:
i pushed it already !!